docs/admin-manual/library.md

```diff
@@ -13,4 +13,14 @@ To scan a directory and import the files into the library, you can use the follo
 sudo -u libretime libretime-api bulk_import --path PATH_THE_DIRECTORY_TO_SCAN
 ```
 
-See the command usage to get available options.
+If you're running from a docker environment, use
+```bash
+docker-compose run --rm api libretime-api bulk_import --path PATH_THE_DIRECTORY_TO_SCAN
```

Comment:
```suggestion
docker-compose run --rm -e LIBRETIME_GENERAL_PUBLIC_URL=http://nginx:8080 api libretime-api bulk_import --path PATH_THE_DIRECTORY_TO_SCAN
```

Ideally I would move the import process to python, but this might do the trick for now.